app/validators/json_validator.rb

```diff
@@ -0,0 +1,8 @@
+class JsonValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    return if value.empty?
+    JSON.parse(value) if value.is_a? String
+  rescue JSON::ParserError
+    record.errors.add(attribute, value_must_be_valid_json) if value.is_a? String
```

Comment:
I think a `:` is missing in front of `value_must_be_valid_json`

Reply:
I renamed it but make a mistake. The next comment is a side-effect. :confounded: